armi/physics/fuelCycle/tests/test_fuelHandlers.py

```diff
@@ -498,7 +498,7 @@ def test_repeatShuffles(self):
         ensure repeatability.
         """
         # check labels before shuffling:
-        for a in self.r.excore["sfp"].getChildren():
+        for a in self.r.excore["sfp"]:
```

Comment:
This called `iterChildren()` under the hood now, right?

Reply:
Yes / no. `getChildren` uses `iterChildren` but if you're just trying to iterate over the immediate children of composite, using `Composite.__iter__` works.